cartridges/templates/wso2as-5.2.1/configs.ini

```diff
@@ -1,6 +1,6 @@
 [SETTINGS]
 READ_FROM_ENVIRONMENT=true
-REPOSITORY_CONF_DIRECTORY=/path/to/wso2as-5.2.1/repository/conf
+REPOSITORY_CONF_DIRECTORY=<AS_HOME>/repository/conf
 
 [PARAMS]
```

Comment:
IMO it would be better to put default configuration that comes with the Carbon product here. Otherwise out of the box the server would fail to start.